<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_CapteurN</class>
 <widget class="QWidget" name="f_CapteurN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icones/oeil-previsualiseur-icone-4524-48.png</normaloff>:/Icones/oeil-previsualiseur-icone-4524-48.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Icones/metal-2.bmp);</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QLabel" name="LbNomCarteN">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>230</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>BOUTTON POUSSOIR VERT 
 CONNECTÉ À LA BROCHE N° 25 </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_ImgCapteurN">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>102</width>
     <height>69</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>69</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnBoutonON">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>110</y>
     <width>70</width>
     <height>95</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Supervision.qrc">
     <normaloff>:/Images/Sup/OFF.png</normaloff>:/Images/Sup/OFF.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>95</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Supervision.qrc"/>
 </resources>
 <connections/>
</ui>
